contracts/IVaultHandler.sol

```diff
@@ -229,6 +237,12 @@ abstract contract IVaultHandler is
     TCAPToken = _tcapAddress;
     rewardHandler = IRewardHandler(_rewardHandler);
     treasury = _treasury;
+		uint8 _collateralDecimals = ERC20(_collateralAddress).decimals();
```

Comment:
We can save some gas if we get rid of lines 240-245 by computing `collateralDecimalsAdjustmentFactor` offline and passing it as a parameter in the constructor. 
Any thoughts on this?

Reply:
For this case, I think it's better to read it directly from the contract to avoid errors from calls